systemtest/mauveLoadTest/playlist.xml

```diff
@@ -86,9 +108,8 @@
 		</impls>
 		<platformRequirements>^os.osx</platformRequirements>
 	</test>
-
 	<test>
-		<testCaseName>MauveSingleInvocationLoadTest_HS</testCaseName>
+		<testCaseName>MauveSingleInvocationLoadTest_OpenJ9</testCaseName>
```

Comment:
Same note as above, can collapse into 1 testCaseName, MauveSingleInvocationLoadTest for 11+ (removing <impls>, so targets are generated for all impls).